pkg/splunk/enterprise/clustermanager.go

```diff
@@ -213,6 +223,14 @@ func ApplyClusterManager(ctx context.Context, client splcommon.ControllerClient,
 			cr.Status.TelAppInstalled = true
 		}
 
+		// Reset symbolic links for clusterManager
+		if cr.Status.SmartStoreConfigured {
+			err = resetSymbolicLinks(ctx, client, cr, 1, podExecClient)
+			if err != nil {
+				return result, err
+			}
+		}
+
```

Comment:
I think, if you add this logic at [here](https://github.com/splunk/splunk-operator/pull/1053/files#diff-4fab20ab91ce601b65a8f1961f3809d7429faffe27c2c88c6c08956dc23a0a74R370), we can avoid the CR changes.

Reply:
@sgontla The link seems to be broken. Also the symbolic links are broken every time a pod is restarted for any reason and hence I was thinking we need this logic here. If we modify only where smartstore is modified then the symbolic links might be broken in between?

Reply:
Seems like some issue with the link, anyhow, i was referring the line number 

> 	podExecClient := splutil.GetPodExecClient(c, cr, cmPodName)
	err := CheckIfsmartstoreConfigMapUpdatedToPod(ctx, c, cr, podExecClient)
	if err != nil {
		return err
	}

   <<<<<<<<<<<<< Moving the above code to hear, will avoid the CR changes, and that way no need to track through the SmartStoreConfigured flag >>>>>>>>>>>>>

	err = PushManagerAppsBundle(ctx, c, cr)
	if err == nil {
		scopedLog.Info("Bundle push success")
		cr.Status.BundlePushTracker.NeedToPushManagerApps = false
	}

Reply:
Just thinking through the main issue of softlinks, as the softlinks are no more reliable, wondering if we can just copy the files?

Reply:
Per our discussion, removed the CR status flag and moved the function call to bundle push API